skillserver/echo.go

```diff
@@ -44,6 +50,7 @@ func (this *EchoRequest) GetIntentName() string {
 	return this.GetRequestType()
 }
 
+// Deprecated: Please use the github.com/mikeflynn/go-alexa/customskill package
 func (this *EchoRequest) GetSlotValue(slotName string) (string, error) {
```

Comment:
I didn't add this to `customskill`. I'm not convinced we need these little helper functions.